ocs_ci/ocs/platform_nodes.py

```diff
@@ -28,7 +28,7 @@ class PlatformNodesFactory:
 
     """
     def __init__(self):
-        self.cls_map = {'AWS': AWSNodes, 'vsphere': VMWareNodes, 'aws': AWSNodes}
+        self.cls_map = {'AWS': AWSNodes, 'vsphere': VMWareNodes, 'aws': AWSNodes, 'baremetal': NodesBase}
```

Comment:
@prsurve need to create a new class for baremetal here. Ok with doing that in a separate PR. Please just report an issue to track it

Reply:
Done create an issue #1965

Reply:
thanks